assets/src/components/mapMarkers.tsx

```diff
@@ -466,7 +466,6 @@ export const LocationMarker = ({
     position={[location.latitude, location.longitude]}
     divIconSettings={{
       iconAnchor: new Leaflet.Point(9, 24),
-      iconSize: [24, 18],
```

Comment:
issue: I don't think we want this removed, as changing this will subtlety offset where leaflet places the marker on the map, because leaflet uses this value for layout and translating the object onto the map. Additionally, this changes the "tap target"/"bounding box" of the marker, so if we did make it clickable, potentially only part of it would be clickable as I understand it.

(If somehow/for some reason we changed the leaflet marker icon class to clip things outside of it's bounding box, the marker would be cut off as well)
![image](https://github.com/mbta/skate/assets/6913198/720bcf7a-48c4-4e12-891a-608c22d8c04b)

Reply:
> Hmmm. I don't love that the CSS and the divIcon are both there and have to have the same values.

Agreed 😞

Reply:
But also, the bounding box stuff matters more, and I'll add that back in 😄